app/controllers/admins/rewards_controller.rb

```diff
@@ -39,10 +39,18 @@ def update
       render :edit, notice: e.message
     end
 
+    def remove_category
```

Comment:
this action looks weird. 
We are simply updating a `reward`, so why not simply use the `update` action instead?

Reply:
You are right, it looks weird. Moreover, it's unnecessary -> removed.